external/rsyslog/rsyslog.conf

```diff
@@ -38,10 +72,28 @@ mail.*                                                  -/var/log/maillog
 cron.*                                                  /var/log/cron
 
 # Everybody gets emergency messages
-*.emerg                                                 *
+*.emerg                                                 :omusrmsg:*
 
 # Save news errors of level crit and higher in a special file.
 uucp,news.crit                                          /var/log/spooler
 
 # Save boot messages also to boot.log
 local7.*                                                /var/log/boot.log
+
+
+# ### begin forwarding rule ###
+# The statement between the begin ... end define a SINGLE forwarding
```

Comment:
Good eye, but I'm going to leave that one since it is an issue with the default rsyslog.conf file that ships with the OS.  If someone tries to diff ours against vanilla, they will just see code-related changes.